admin/class-admin-init.php

```diff
@@ -275,6 +276,59 @@ private function get_compatibility_notification() {
 	}
 
 	/**
+	 * Determines whether a suggested plugins notification needs to be displayed.
+	 *
+	 * @return void
+	 */
+	public function yoast_plugin_suggestions_notification() {
+		$checker = new WPSEO_Plugin_Availability();
+		$notification_center = Yoast_Notification_Center::get();
+
+		// Get all Yoast plugins that have dependencies.
+		$plugins = $checker->get_plugins_with_dependencies();
+
+		foreach ( $plugins as $plugin_name => $plugin ) {
+			$dependency_name = $checker->get_dependency_names( $plugin )[0];
```

Comment:
Array dereferencing the result of a function only became available in PHP 5.4. This line needs to be rewritten.
Ref: http://php.net/manual/en/language.types.array.php#language.types.array.syntax.accessing